go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/apache/skywalking-eyes v0.4.0
```

Comment:
Please run ```make bazel_prepare``` to sync gomod info to bazel.

Reply:
Got error when running `make bazel_prepare`:
```
Error parsing module for more error information: json: cannot unmarshal string into Go struct field moduleFromDownload.Error of type golang.downloadError
make: *** [bazel_prepare] Error 1
```

Reply:
Done!